bc_obps/registration/api/user_operator.py

```diff
@@ -286,9 +309,10 @@ def create_user_operator_contact(request, payload: UserOperatorContactIn):
 
 ##### PUT #####
 
-
+# Andrea
 @router.put("/select-operator/user-operator/{user_id}/update-status")
 def update_user_operator_status(request, user_id: str):
+    raise_401_if_role_not_authorized(request, ["cas_admin", "cas_analyst"])
```

Comment:
I think we'll also need to add "industry_user"admin" to the list of permitted roles, to accommodate for when an industry user submits a request for access to an operator that already has an industry_user_admin connected to it